### DISCUSSION TITLE
Which is the difference between super.onCreate(savedInstanceState) and super.onCreate(null)?

### DISCUSSION THREAD

**Discussion by sebadiaz-arg** [software-mansion/react-native-screens#1567] [Q&A] [answered]
Hi! in our app we saw some errors related with the use of super.onCreate(savedInstanceState), as we has that line defined. Looking here into the resolved issues we have found as recommendation to use super.onCreate(null) instead of. But, we don't understand what would produce this change in our app. Can someone clarify about it? Thanks in advance.

**Comment by kkafar** [comment, marked as answer]
Hi @sebadiaz-arg, I encourage you to read through comments from this issue: https://github.com/software-mansion/react-native-screens/issues/17
